samples/batch/csharp/packages.config

```diff
@@ -1,5 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We probably should switch to package references.

Reply:
done.